doc/nrf/ug_nrf52.rst

```diff
@@ -122,6 +122,20 @@ In addition, you can run the :ref:`zephyr:bluetooth-samples` that are included f
 
 For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).
 
+Bluetooth mesh
+==============
+
+Bluetooth mesh is a low-power mesh networking solution based on Bluetooth Low Energy, optimized for creating large-scale device networks.
```

Comment:
Is mesh a "low-power" solution? Maybe we should remove "low-power"

Reply:
Sentences like "Unlike other low-power wireless mesh networking technologies, Bluetooth mesh is..." and "Why should I choose Bluetooth mesh networking over other low-power wireless mesh solutions" used on Bluetooth's webpage make me definitely think that Bluetooth mesh is considered a low-power solution. But I will remove low-power if that is the safer choice.